src/Formidable.ml

```diff
@@ -366,8 +366,13 @@ module Make
       let children = props##children in
       let value = Optic.Lens.view props##lens values in
       let isFocused, setIsFocused = React.useState (fun () -> false) in
+
       let validationNames =
-        Option.fold [] (List.map Validations.getName) props##validations
+        Option.fold []  (List.flatMap (Validations.getName >> Relude.String.splitAsList ~delimiter:"<or>")) props##validations
```

Comment:
Ah sorry, for that one, I thought we could have a list of string instead of a string in the Validations

So in the Validations we have:

```ocaml
module Pair = struct
  type ('values, 'value, 'error) t =
    string * ('values, 'value, 'error) Validator.t
end

type ('values, 'value, 'error) t = Strategy.t * ('values, 'value, 'error) Pair.t
```

and it could be

```ocaml
module Pair = struct
  type ('values, 'value, 'error) t =
    (string list) * ('values, 'value, 'error) Validator.t
end

type ('values, 'value, 'error) t = Strategy.t * ('values, 'value, 'error) Pair.t
```